uber/templates/preregistration/dealer_registration.html

```diff
@@ -1,10 +0,0 @@
-{% extends "./preregistration/preregbase.html" %}
-{% block title %}Preregistration{% endblock %}
-{% block backlink %}{% endblock %}
-{% block content %}
-
```

Comment:
I don't remember offhand why I deleted this; I probably saw that it wasn't being linked to anywhere and assumed it was a vestigial file which everyone had forgotten about.  We can definitely add it back in, yeah; I just created a ticket for that: https://github.com/magfest/ubersystem/issues/1101